core/includes/database/database.inc

```diff
@@ -3094,18 +3104,26 @@ function db_change_field($table, $field, $field_new, $spec, $keys_new = array())
  */
 
 /**
- * Sets a session variable specifying the lag time for ignoring a slave server.
+ * Sets a session variable specifying the lag time for ignoring a replica server.
  */
-function db_ignore_slave() {
+function db_ignore_replica() {
   $connection_info = Database::getConnectionInfo();
-  // Only set ignore_slave_server if there are slave servers being used, which
+  // Only set ignore_replica_server if there are replica servers being used, which
   // is assumed if there are more than one.
   if (count($connection_info) > 1) {
-    // Five minutes is long enough to allow the slave to break and resume
+    // Five minutes is long enough to allow the replica to break and resume
     // interrupted replication without causing problems on the Backdrop site
     // from the old data.
     $duration = settings_get('maximum_replication_lag', 300);
-    // Set session variable with amount of time to delay before using slave.
-    $_SESSION['ignore_slave_server'] = REQUEST_TIME + $duration;
+    // Set session variable with amount of time to delay before using replica.
+    $_SESSION['ignore_replica_server'] = REQUEST_TIME + $duration;
   }
 }
+
+/**
+ * Backwards-compatible wrapper function.
+ *   @todo Remove in Backdrop 2.0.
```

Comment:
This should be `@deprecated since Backdrop 1.17.0` rather than `@todo`. Ideally all deprecated code would be removed in Backdrop 2.0.